chart/templates/deployment.yaml

```
@@ -21,10 +21,57 @@ spec:
       imagePullSecrets:
         {{- toYaml .Values.imagePullSecrets | nindent 8 }}
       {{- end }}
+      {{- if eq .Values.config.OBOT_SERVER_ENCRYPTION_PROVIDER "custom" }}
+      initContainers:
+        - name: encryptionsetup
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: OBOT_SERVER_ENCRYPTION_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: OBOT_SERVER_ENCRYPTION_KEY
+                  name: {{ include "obot.config.secretName" . }}
+          command:
+          - /bin/sh
+          - -c
+          - |
+            cat > /config/encryption.yaml <<EOF
+            kind: EncryptionConfiguration
+            apiVersion: apiserver.config.k8s.io/v1
+            resources:
+              - resources:
+                  - credentials
+                  - runstates.obot.obot.ai
+                  - users.obot.obot.ai
+                  - identities.obot.obot.ai
```

There's one more new type that this needs now.

```suggestion
                  - credentials
                  - runstates.obot.obot.ai
                  - users.obot.obot.ai
                  - identities.obot.obot.ai
                  - mcpoauthtokens.obot.obot.ai
```